networkit/cpp/components/test/ConnectedComponentsGTest.cpp

```diff
@@ -93,6 +93,22 @@ TEST_F(ConnectedComponentsGTest, testParallelConnectedComponents) {
     }
 }
 
+TEST_F(ConnectedComponentsGTest, testParallelConnectedComponents2) {
+    Graph G(6);
+    G.addEdge(0, 1);
+    G.addEdge(1, 2);
+    G.addEdge(3, 4);
+    G.addEdge(4, 5);
+    ConnectedComponents cc(G);
+    cc.run();
+    count seqNum = cc.numberOfComponents();
+    ParallelConnectedComponents ccPar(G);
+    ccPar.run();
+    count parNum = ccPar.numberOfComponents();
+    EXPECT_EQ(seqNum, parNum);
```

Comment:
```suggestion
    ParallelConnectedComponents ccPar(G);
    ccPar.run();
    EXPECT_EQ( cc.numberOfComponents(), ccPar.numberOfComponents());
```

Could you also:
- add another test with a bigger graph from `input`
- check that the components computed by `ParallelConnectedComponents` in the next test are the same as those computed by `ConnectedComponents` (as you are already doing below).